project/GraalVM.scala

```diff
@@ -259,7 +260,7 @@ object GraalVM {
     val vmVersion = System.getProperty("java.vm.version")
     tryParseJavaVMVersion(vmVersion) match {
       case Some(version) =>
-        if (version != graalVersion) {
+        if (!isSameVersion(version, graalVersion)) {
           log.error(
             s"Running on GraalVM version $version. " +
             s"Expected GraalVM version $graalVersion."
```

Comment:
I've just tried to run on Windows with wrong GraalVM:
```
[info] set current project to enso (in build file:/S:/windows/enso/)
[error] Running on Java version 24. Expected Java version 25.
[warn] there's a key that's not used by any other settings/tasks:
[warn]
[warn] * ThisBuild / additionalOpts
[warn]   +- S:\windows\enso\build.sbt:6272
[warn]
[warn] note: a setting might still be used by a command; to exclude a key from this `lintUnused` check
[warn] either append it to `Global / excludeLintKeys` or call .withRank(KeyRanks.Invisible) on the key
[warn] Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore? (default: r)
```
Can the error message _"Running on Java version 24. Expected Java version 25."_ include a download link, please?